inc/namespace.php

```diff
@@ -1084,3 +1086,32 @@ function get_logger( string $log_group, string $log_stream ) : LoggerInterface {
 	$loggers[ $tag_name ] = $logger;
 	return $loggers[ $tag_name ];
 }
+
+/**
+ * Register the "Debug Request" menu item.
+ *
+ * @param WP_Admin_Bar $wp_admin_bar Admin bar instance.
+ */
+function register_trace_menu_item( WP_Admin_Bar $wp_admin_bar ) {
+	if ( ! defined( 'HM_ENV_REGION' ) ) {
```

Comment:
This isn't really sufficient to check for a cloud environment, there's an `is_cloud()` function in this namespace.

Reply:
I did wonder, but this is the same check as in the environment indicator code:

https://github.com/humanmade/altis-cloud/blob/deac1cc0d8aeeda006d8ccc38f136330596bf5d8/inc/environment_indicator/namespace.php#L93-L100

Reply:
Hmm. That's a little different to be fair, it can still be useful to jump to the dashboard from your local, but not to jump to a specific xray entry in the dashboard from your local.